code/game/objects/items/rcd/RPD.dm

```diff
@@ -614,14 +636,19 @@ GLOBAL_LIST_INIT(transit_tube_recipes, list(
 							tube.wrench_act(user, src)
 					return
 			else
-				return ..()
+				return
 
 /obj/item/pipe_dispenser/attackby(obj/item/item, mob/user, params)
 	if(istype(item, /obj/item/rpd_upgrade))
 		install_upgrade(item, user)
 		return TRUE
 	return ..()
 
+/obj/item/pipe_dispenser/afterattack(obj/attacked_object, mob/living/user, adjacent, params)
+	if(upgrade_flags & RPD_UPGRADE_BLUESPACE)
+		rpd_attack_action(attacked_object, user)
+	return ..()
```

Comment:
Yeah this needs at the very least a view range limit, otherwise you're allowing people to remotely place pipes through cameras on the opposite side of the map. No thanks